android/PhysicalWeb/app/build.gradle

```diff
@@ -4,7 +4,7 @@ apply plugin: 'pmd'
 
 android {
     compileSdkVersion 25
-    buildToolsVersion "25.0.0"
+    buildToolsVersion "25.0.1"
```

Comment:
(I'm curious as to whether or not travis will build with this...we might have to change values until it passes)